org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/JavaSearchScope.java

```diff
@@ -661,10 +659,9 @@ public String toString() {
 	StringBuilder result = new StringBuilder("JavaSearchScope on "); //$NON-NLS-1$
 	if (this.elements != null) {
 		result.append("["); //$NON-NLS-1$
-		for (int i = 0, length = this.elements.size(); i < length; i++) {
-			JavaElement element = (JavaElement)this.elements.get(i);
+		for (Object element : this.elements) {
```

Comment:
```suggestion
		for (JavaElement element : this.elements) {
```

Reply:
I followed the existing style as closely as I could to create the minimal change.